filament/backend/include/backend/DriverEnums.h

```diff
@@ -369,6 +369,18 @@ enum class CompressedPixelDataType : uint16_t {
     DXT1_RGB, DXT1_RGBA, DXT3_RGBA, DXT5_RGBA,
     DXT1_SRGB, DXT1_SRGBA, DXT3_SRGBA, DXT5_SRGBA,
 
+    // RGTC formats available with a GLES extension
+    RED_RGTC1,              // BC4 unsigned
+    SIGNED_RED_RGTC1,       // BC4 signed
+    RED_GREEN_RGTC2,        // BC5 unsigned
+    SIGNED_RED_GREEN_RGTC2, // BC5 signed
+
+    // BPTC formats available with a GLES extension
+    RGB_BPTC_SIGNED_FLOAT,  // BC6H signed
+    RGB_BPTC_UNSIGNED_FLOAT,// BC6H unsigned
+    RGBA_BPTC_UNORM,        // BC7
+    SRGB_ALPHA_BPTC_UNORM,  // BC7 sRGB
+
```

Comment:
Move these at the end of the list so that it doesn't change the enum values of existing formats.